docs/tutorial.md

```diff
@@ -544,12 +505,6 @@ Now we have a `createOrUpdateResource` method. It calls
 `createMemcachedDeployment` which we have implemented above. In the next section
 we will discuss the deletion of the resource.
```

Comment:
This line regarding the "deletion" section needs to go as well.

Reply:
The sentence under " labelsForMemcached" section:
> Below the `deleteResource` method add the following
helper:

also needs to be removed

Reply:
Updated as `createOrUpdateResource`